.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - run: yarn install --immutable --immutable-cache --check-cache
       - run: yarn lint
       - run: yarn build
-      - run: yarn test
+      - run: yarn test --passWithNoTests
```

Comment:
why?

Reply:
tests are empty. build error when pusing the code.